src/clojure_tic_tac_toe/game_handler.clj

```diff
@@ -1,13 +1,16 @@
 (ns clojure-tic-tac-toe.game_handler
   (:require [clojure-tic-tac-toe.board :as board]
+            [clojure-tic-tac-toe.console_ui.input_output :as io]
```

Comment:
The `game_handler` should not depend on anything ui/io.  This is the core logic you should be able to use in any type of ui.

Reply:
Moved `io` function calls to `console_ui`